src/miral/static_display_config.cpp

```diff
@@ -638,3 +655,15 @@ void miral::ReloadingYamlFileDisplayConfig::auto_reload()
     }
 }
 
+void miral::ReloadingYamlFileDisplayConfig::check_for_layout_override()
+{
+    std::lock_guard lock{mutex};
+    if (!config_path_) return;
+
+    if (std::ifstream layout_file{config_path_.value() + "/" + basename + "-layout"})
```

Comment:
The `"-layout"` string literal only appears twice, but still seems like it would be a better idea to not repeat it, either by putting it in a const global, a `layout_override_for_basename(basename)` standalone helper or something else.